layers/+spacemacs/spacemacs-editing-visual/packages.el

```diff
@@ -123,13 +125,13 @@
       (setq indent-guide-delay 0.3)
       (spacemacs|add-toggle indent-guide
         :mode indent-guide-mode
-        :documentation
-        "Highlight indentation level at point. (alternative to highlight-indentation)."
+        :documentation "Highlight indentation level at point.
+                        (alternative to highlight-indentation)."
```

Comment:
This doesen't seem right, it should be two concatenated strings otherwise the trailing whitespaces in the second line will be displayed as well.